modules/auxiliary/scanner/winrm/winrm_login.rb

```diff
@@ -50,20 +50,21 @@ def run_host(ip)
     )
 
     kerberos_authenticator_factory = nil
+    # TODO: This is Marked as dead for some reason
     preferred_auth = 'Negotiate'
```

Comment:
In other places this is `self.preferred_auth` - will have to verify if this is just copy/paste that's been left behind or if it's meant to be set

Reply:
Dead code; Nuked in a separate PR